man/fi_info.1.md

```diff
@@ -92,28 +92,184 @@ the fi_info structure, see fi_getinfo(3).
 # USAGE EXAMPLES
 
 ```
-$ fi_info -n 30.0.11.1 -p usnic -t FI_EP_DGRAM
+$ fi_info -p verbs -t FI_EP_DGRAM
```

Comment:
In the other cases, you add backslash before dash. but not here.  I don't think the backslash is needed, since we're in quoted text.